src/containers/Tag/TagAdd/TagAdd.tsx

```diff
@@ -25,9 +31,44 @@ export const TagAdd: React.SFC<TagAddProps> = (props: TagAddProps) => {
 	const [parentId, setParentId] = useState(tag ? tag.parent_id : "");
 	const [formSubmitted, setFormSubmitted] = useState(false);
 
+	const [nameError, setNameError] = useState(false);
```

Comment:
I am not sure if this is a scalable approach and we need to make it more generic. Ideally, all the validations should be done in a helper function which can be used in multiple forms.